rag/rag-mdfiles/frontend/.gitignore

```diff
@@ -0,0 +1,40 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
same here

Reply:
This is a next js frontend. The .gitignore at the root of the repo is not done for next.js or next.js in nested folder.
We can have it at root level if we think a lot of the recipes will have next.js parts.
@johnymontana do you want to ignore files on possible nextjs folders in the recipes at root level?

Reply:
@rderbier yes, that sounds good. I think we will end up with several next.js projects in the recipes going forward (I'm working on one for the neo4j recipe now)

Reply:
@ryanfoxtyler @rderbier actually after rethinking this one, I think it would make more sense to keep the .gitignore files in the subdirectories because if a user goes to deploy or run the individual recipe or copies it out of the subdirectory we want all relevant files at that level. If we move .gitignores to the root of this repo then they would be missing that file.

Reply:
I can get on board with that, but it would be one per recipe (not 3), right?